### DISCUSSION TITLE
Large Amount of Errors in Usage Dashboard but Nothing in Logs

### DISCUSSION THREAD

**Discussion by jadamconnor** [vercel/community#301] [Help] [answered]
The vast majority (97% yesterday) of my serverless functions have errored according to my usage dashboard, but I'm not seeing anything in logs. Nor are there any issues even approaching this number in Sentry.io. Anyone have any idea what might be causing this?

**Comment by corygibbons** [comment, marked as answer]
Myself and others experiencing the same problem (vercel/community#289). However as of a couple hours ago it seems to have been resolved in my projects 🤞

**Reply by jadamconnor** [reply]
> Myself and others experiencing the same problem (vercel/community#289). However as of a couple hours ago it seems to have been resolved in my projects 🤞

Thanks for the heads up! I was concerned. Looks like a platform issue though.

**Reply by leerob** [reply]
We are currently investigating this 👍 Let's continue the conversation in #289 - thank you!
